Sources/JobsPostgres/PostgresJobsQueue.swift

```diff
@@ -362,26 +373,42 @@ public final class PostgresJobQueue: JobQueueDriver {
 
     func setStatus(jobID: JobID, status: Status, connection: PostgresConnection) async throws {
         try await connection.query(
-            "UPDATE _hb_pg_jobs SET status = \(status), lastModified = \(Date.now) WHERE id = \(jobID)",
+            """
+            UPDATE swift_jobs.jobs
+            SET status = \(status),
+                last_modified = \(Date.now)
+            WHERE id = \(jobID) AND queue_name = \(configuration.queueName)
+            """,
             logger: self.logger
         )
     }
 
     func setStatus(jobID: JobID, status: Status) async throws {
         try await self.client.query(
-            "UPDATE _hb_pg_jobs SET status = \(status), lastModified = \(Date.now) WHERE id = \(jobID)",
+            """
+            UPDATE swift_jobs.jobs
+            SET status = \(status),
+                last_modified = \(Date.now)
+            WHERE id = \(jobID) AND queue_name = \(configuration.queueName)
+            """,
             logger: self.logger
         )
     }
 
-    func getJobs(withStatus status: Status) async throws -> [JobID] {
+    func getJobs(withStatus status: Status) async throws -> [(id: JobID, queue: String)] {
         let stream = try await self.client.query(
-            "SELECT id FROM _hb_pg_jobs WHERE status = \(status) FOR UPDATE SKIP LOCKED",
+            """
+            SELECT
+                id,
+                queue_name
+            FROM swift_jobs.jobs
+            WHERE status = \(status) AND queue_name = \(configuration.queueName)
+            """,
```

Comment:
We don't need to get the queue name here. It is always `configuration.queueName`